samples/tenants/webapp/localService/base.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const fs = require('fs');
+
+module.exports = {
+    /**
+     * This function returns the initial dataset for the WorkstationSet entity.
+     * It reads the content from the corresponding JSON file and returns it as an array of objects.
+     * @param {string} contextId - Represents the tenant id for the current request (optional).
+     * @param {string} fileName - The name of the file to load with full path.
+     * @param {string} currentDir - The current directory of the file.
+     * @returns {Array} - The initial dataset for the WorkstationSet entity.
+     */
+    getInitialDataSet: function (contextId, fileName, currentDir) {
+        let filePath = '';
+
+        // Extract the base file name without extension
+        const baseFileName = path.basename(fileName, path.extname(fileName)).replace(/\.js$/, '');
```

Comment:
I get the idea but for a sample it seems overkill.
I did however, consider including this in the mockserver by default, just have to come up with a proper naming / folder organization for that.